css/style.css

```diff
@@ -57,6 +56,96 @@ body {
   display: none;
 }
 
+.mobile-menu {
+  display: none;
+  position: absolute;
+  left: 0;
+  top: 0;
+  z-index: 1;
+  width: 375px;
+  height: 812px;
+  background: #fff;
+  bottom: 57px;
+  box-shadow: 0 48px 48px rgb(37, 47, 137, 0.08);
+}
```

Comment:
- [ ] As requested by the previous kindly make sure your designs look like the provided Figma design. :wink: 

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/201733901-472c0b4a-eb82-4f6c-bc66-e2cd4690d88f.png)


  - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/201734371-08675244-3d3c-4119-80a4-e711eeac6385.png)

Reply:
@levy002 Good evening; 
Thanks for your comment and support, could you give me please some clue what I will make changes, I did not understand you! Kind regards! Would you like actually to add the border-radius or not?